README.md

```diff
@@ -16,8 +16,13 @@ specification](#custom-tuning-specification) is passed to all
 daemons running in the cluster in the format that the daemons understand.
 The daemons run on all nodes in the cluster, one per node.
 
+Node-level settings applied by the containerized tuned are rolled back on
+an event that triggers a profile change or when the containerized tuned
+daemon is terminated gracefully by receiving and handling a termination
+signal.
 
```

Comment:
terminated gracefully by "sending a termination signal"

Reply:
Sending?  The containerized tuned daemon itself doesn't send any termination signals.  It *receives* and *handles* termination signals.  openshift-tuned though does send a `TERM` signal to tuned when it receives a `TERM` signal itself.